conf/messages

```diff
@@ -199,18 +199,8 @@ tariff.declaration.type.title = More details about standard declarations
 tariff.declaration.type.text.1 = A standard declaration (arrived or pre-lodged) is a full declaration. This means all sections of the declaration must be completed.
 tariff.declaration.type.text.2 = Find details about {0}.
 tariff.declaration.type.linkText.2 = making a full export declaration (opens in a new tab)
```

Comment:
Are you sure you still need lines 198-201 and 203?

Reply:
Removed unused messages, few still in use.